charts/lws/README.md

```diff
@@ -0,0 +1,67 @@
+# lws's helm chart
+
+## Table of Contents
+
+<!-- toc -->
+- [Installation](#installation)
+    - [Prerequisites](#prerequisites)
+    - [Installing the chart](#installing-the-chart)
+        - [Install chart using Helm v3.0+](#install-chart-using-helm-v30)
+        - [Verify that controller pods are running properly.](#verify-that-controller-pods-are-running-properly)
+    - [Configuration](#configuration)
+<!-- /toc -->
+
+### Installation
+
+Quick start instructions for the setup and configuration of lws using Helm.
+
+#### Prerequisites
+
+- [Helm](https://helm.sh/docs/intro/quickstart/#install-helm)
+
+#### Installing the chart
+
+##### Install chart using Helm v3.0+
+
+```bash
+$ git clone git@github.com:kubernetes-sigs/lws.git
+$ cd charts
+$ helm install lws lws --create-namespace --namespace lws-system
+```
```

Comment:
how do we install this from a release package? can we add instructions to making the helm charts an artifact of the release?

Reply:
Maybe we can store the helm packages in github with gh-pages. But that's the next step.

Reply:
Installing lws from a release package sounds good, and I agree that it's a step we consider next.

Reply:
Install in a different namespace is another thing I'm concerned, but that can be follow up together.

Reply:
can you please open issues to track the follow up items?

Reply:
I created here https://github.com/kubernetes-sigs/lws/issues/289